src/features/query-bar/__tests__/Condition-test.js

```diff
@@ -80,31 +78,40 @@ describe('features/query-bar/components/filter/Condition', () => {
         const index = 0;
         const condition = initialCondition;
         const dateFieldValue = new Date(2018, 11, 24, 10, 33, 30, 0);
-        const fieldId = undefined;
         const valueType = 'string';
         const keyType = VALUE_KEY;
         const displayTextType = 'valueDisplayText';
 
         test.each`
-            description                                                | fieldValue          | displayText       | value
-            ${'user has entered an empty string into the value field'} | ${stringFieldValue} | ${''}             | ${''}
-            ${'user has selected a date in the date picker'}           | ${dateFieldValue}   | ${dateFieldValue} | ${dateFieldValue}
+            description                                            | fieldValue          | displayText       | value
+            ${'should enter an empty string into the value field'} | ${stringFieldValue} | ${''}             | ${''}
+            ${'should select a date in the date picker'}           | ${dateFieldValue}   | ${dateFieldValue} | ${dateFieldValue}
         `('$description', ({ fieldValue, displayText, value }) => {
-            const update = jest.fn();
-            const wrapper = getWrapper({ update });
+            const onFieldChange = jest.fn();
+            const wrapper = getWrapper({ onFieldChange });
 
             wrapper.find('ValueField').prop('updateValueField')(fieldValue);
 
-            expect(update).toHaveBeenCalledWith(
+            expect(onFieldChange).toHaveBeenCalledWith(
                 index,
                 condition,
                 displayText,
                 displayTextType,
-                fieldId,
                 value,
                 keyType,
                 valueType,
             );
         });
     });
+
+    describe('getColumnOptions()', () => {
+        test.each`
+            description                                             | expectedColumnOptions
+            ${'should open the value dropdown and see the options'} | ${columnOptions}
+        `('$description', ({ expectedColumnOptions }) => {
+            const wrapper = getWrapper({});
```

Comment:
Is expectedColumnOptions somehow implicitly passed to the component? I think it should be explicit here.